aws/table_aws_oam_sink.go

```diff
@@ -0,0 +1,195 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/oam"
+	"github.com/aws/aws-sdk-go-v2/service/oam/types"
+)
+
+func tableAwsOAMSink(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_oam_sink",
+		Description: "AWS OAM Sink",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("arn"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterValue", "ResourceNotFoundException"}),
+			},
+			Hydrate: getAwsOamSink,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsOamSinks,
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the sink.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "The random ID string that Amazon Web Services generated as part of the sink ARN.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "arn",
+				Description: "The ARN of the sink.",
+				Type:        proto.ColumnType_STRING,
+			},
+
+			/// Standard columns for all tables
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "tags",
```

Comment:
Don't we need the `tag_src` column?

Reply:
We don't need `tag_src` column as we are getting response in `map[string]string` type.